internal/app/api/handler/login/login.go

```diff
@@ -50,3 +65,101 @@ func (h *LoginHandler) GetUser(w http.ResponseWriter, r *http.Request) {
 	respond.JSON(w, http.StatusAccepted, u)
 	return
 }
+func (h *LoginHandler) Register(w http.ResponseWriter, r *http.Request) {
+	username := r.FormValue("username")
+	rawPassword := r.FormValue("password")
+	fullName := r.FormValue("fullname")
+	email := r.FormValue("email")
+	deliveryCenter := r.FormValue("deliverycenter")
+	dcs := []string{deliveryCenter}
+	bytes, err := bcrypt.GenerateFromPassword([]byte(rawPassword), 14)
```

Comment:
This kind of thing should be handled in service layer in set password API. Handler should call only the AddUser api here...